generators/rails-generator/rbs_collection.yaml

```diff
@@ -0,0 +1,94 @@
+# Download sources
+sources:
+  - name: ruby/gem_rbs_collection
+    remote: https://github.com/ruby/gem_rbs_collection.git
+    revision: main
+    repo_dir: gems
+
+# A directory to install the downloaded RBSs
+path: .gem_rbs_collection
+
+gems:
```

Comment:
I think it is not the intended usage of `rbs collection`.

`rbs collection` is designed to manage RBS dependencies. It supports downloading and loading RBSs to use them by Steep and so on.

I think this `rbs_collection.yaml` should be used to develop this generator. Actually, I misunderstood this file is used for it.


This usage introduces impedance mismatch, such as the many ignored gems.

So, I'd like to avoid using `rbs_collection.yaml` for this purpose.
How about the following alternative approach?


* List library names by hands without `rbs collection`.
* Rename this file like `rbs_collection-all-stdlibs.yaml`
* Create a directory and put `Gemfile` and `rbs_collection.yaml` to load all libraries.
  * e.g. `mkdir all-stdlibs` and put a Gemfile that contains nokogiri, rack, and so on.

Reply:
It is mainly used by orthoses to check for overlap with existing definitions.
Renamed to orthoses.yaml.